uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/test/manager-leave/components/manager-leave-requests_test.js

```diff
@@ -147,8 +149,17 @@
             expect(controller.levelTypes).toEqual(optionGroupMock.getCollection('hrjc_level_type'));
           });
 
-          it('absence periods have loaded', function () {
-            expect(controller.absencePeriods.length).not.toBe(0);
+          describe('absence periods', function() {
+            it('absence periods have loaded', function () {
+              expect(controller.absencePeriods.length).not.toBe(0);
```

Comment:
are you using mocked data for the absence periods? if yes, can you have a more meaningful test then "the amount is not zero"? ideally you should test that the controller has the exact amount of periods that you expect it to have